datasets/coached_conv_pref/README.md

```diff
@@ -46,78 +50,194 @@ task_ids:
+* `text`: The raw text as written by the ASSISTANT, or transcribed from the spoken recording of USER.
+* `segments`: An array of semantic annotations of spans in the text.  
 
 Each semantic annotation segment has the following fields:
 
-* startIndex: The position of the start of the annotation in the utterance text.
-* endIndex: The position of the end of the annotation in the utterance text.
-* text: The raw text that has been annotated.
-* annotations: An array of annotation details for this segment.
+* `startIndex`: The position of the start of the annotation in the utterance text.
+* `endIndex`: The position of the end of the annotation in the utterance text.
+* `text`: The raw text that has been annotated.
+* `annotations`: An array of annotation details for this segment.
 
 Each annotation has two fields:
 
-* annotationType: The class of annotation (see ontology below).
-* entityType: The class of the entity to which the text refers (see ontology below).
+* `annotationType`: The class of annotation (see ontology below).
+* `entityType`: The class of the entity to which the text refers (see ontology below).
 
 **EXPLANATION OF ONTOLOGY**
 
 In the corpus, preferences and the entities that these preferences refer to are annotated with an annotation type as well as an entity type.
 
 Annotation types fall into four categories:
 
-* ENTITY_NAME: These mark the names of relevant entities mentioned.
-* ENTITY_PREFERENCE: These are defined as statements indicating that the dialog participant does or does not like the relevant entity in general, or that they do or do not like some aspect of the entity. This may also be thought of the participant having some sentiment about what is being discussed.
-* ENTITY_DESCRIPTION: Neutral descriptions that describe an entity but do not convey an explicit liking or disliking.
-* ENTITY_OTHER: Other relevant statements about an entity that convey relevant information of how the participant relates to the entity but do not provide a sentiment. Most often, these relate to whether a participant has seen a particular movie, or knows a lot about a given entity.
+* `ENTITY_NAME`: These mark the names of relevant entities mentioned.
+* `ENTITY_PREFERENCE`: These are defined as statements indicating that the dialog participant does or does not like the relevant entity in general, or that they do or do not like some aspect of the entity. This may also be thought of the participant having some sentiment about what is being discussed.
+* `ENTITY_DESCRIPTION`: Neutral descriptions that describe an entity but do not convey an explicit liking or disliking.
+* `ENTITY_OTHER`: Other relevant statements about an entity that convey relevant information of how the participant relates to the entity but do not provide a sentiment. Most often, these relate to whether a participant has seen a particular movie, or knows a lot about a given entity.
```

Comment:
Since the categories are stored as integers in the examples (ClassLabel), maybe mention the correspondence explicitly ?

```suggestion
* `ENTITY_NAME` (0): These mark the names of relevant entities mentioned.
* `ENTITY_PREFERENCE` (1): These are defined as statements indicating that the dialog participant does or does not like the relevant entity in general, or that they do or do not like some aspect of the entity. This may also be thought of the participant having some sentiment about what is being discussed.
* `ENTITY_DESCRIPTION` (2): Neutral descriptions that describe an entity but do not convey an explicit liking or disliking.
* `ENTITY_OTHER` (3): Other relevant statements about an entity that convey relevant information of how the participant relates to the entity but do not provide a sentiment. Most often, these relate to whether a participant has seen a particular movie, or knows a lot about a given entity.
```

Reply:
Good suggestion.

Made the changes.